doc/toolbox-run.1.md

```diff
@@ -13,13 +13,9 @@ toolbox\-run - Run a command in an existing toolbox container
 Runs a command inside an existing toolbox container. The container should have
 been created using the `toolbox create` command.
 
-`toolbox run` wraps around `podman exec` and by default passes several options
-to it. It allocates a tty, connects to stdin, runs the passed command as the
-current user in the current directory and shares common environmental
-variables.
-
-The executed command is wrapped in `capsh` that gets rid of all extra
-capabilities that could negatively affect the experience.
+On Fedora, the default container is known as `fedora-toolbox-N`, where N is
```

Comment:
This part is already mentioned in `tooöbox-create(1)`. Is it necessary to mention it here, too?

Reply:
It's mentioned in `toolbox-create(1)` from the point of view of `create` - how the name of the image maps to that of the container. For a reader new to Toolbox that might still be a bit of a mental exercise.

A user will encounter the default naming scheme for containers very often, and the concept of a *default container* is very pervasive in Toolbox. So it's important to clearly explain it in the context of creating and using a container.

Generally, for technical documentation it's sometimes good to repeat the same thing if it's sufficiently important. Either to refresh the user's memory or to draw their attention to it. Having to traverse too many references can get disorienting.

Reply:
Very true. Thank you for the explanation!